public/components/overview/components/dashboard_controls.tsx

```diff
@@ -48,15 +50,29 @@ export function DashboardControls() {
   };
 
   return isDashboardSelected ? (
-    <EuiFlexGroup gutterSize="s" justifyContent="spaceBetween">
-      <EuiFlexItem grow={true}>
-        <EuiText size="s" className="obsOverviewDashboardHeader">
-          <p>
-            <EuiLink onClick={() => redirectToDashboards('/view/' + dashboardState?.dashboardId)}>
-              {dashboardState?.dashboardTitle}
-            </EuiLink>
-          </p>
-        </EuiText>
+    <EuiFlexGroup gutterSize="s" alignItems="center" justifyContent="spaceBetween">
+      <EuiFlexItem grow={false}>
+        <EuiFlexGroup alignItems="center" gutterSize="s">
+          <EuiFlexItem grow={false}>
+            <EuiText size="m" className="obsOverviewDashboardHeader">
+              <h4>{dashboardState?.dashboardTitle}</h4>
+            </EuiText>
+          </EuiFlexItem>
+          <EuiFlexItem grow={false}>
+            <EuiToolTip
+              content={
+                <FormattedMessage id="dashboard.popout.tooltip" defaultMessage="Go to dashboard" />
```

Comment:
Should we use `observability.dashboard.popout.tooltip`? dashboard is too generic.

Reply:
Done